packages/react-icons/src/utils/createSvgIconWithRoot.ts

```diff
@@ -0,0 +1,56 @@
+import * as React from 'react';
+// @ts-ignore
+import { css, getNativeProps, htmlElementProperties } from '@uifabric/utilities/src';
+import { classNames, MS_ICON } from 'office-ui-fabric-react/src/components/Icon/SvgIcon.styles';
+import { ISvgIconProps } from 'office-ui-fabric-react/src/components/Icon/SvgIcon.types';
+
+export type SvgIconFuncArg<TProps = ISvgIconProps> = {
+  classNames: { [iconSlot: string]: string }; // renamed from classes
```

Comment:
I'd prefer `classes` as its shorter, but also I had read that `class` was going to make its way back to replace `className` at some point (https://github.com/facebook/react/issues/13525).

Reply:
Let's keep classes then, we will have a closer API with v0